internal/sauceignore/matcher.go

```diff
@@ -36,6 +37,8 @@ func PatternsFromFile(path string) ([]Pattern, error) {
 			ps = append(ps, NewPattern(s))
 		}
 	}
+	// make sure sauce-runner.json is included
```

Comment:
Since this is written from the sauce ignore perspective (its purpose is to ignore files), saying `included` would mean to include the file in its list of ignored files.

Just reducing a bit of ambiguity:
```suggestion
	// make sure that sauce-runner.json is never accidentally ignored by the user
```

Reply:
make sense!